daffodil-test/src/test/resources/org/apache/daffodil/usertests/SepTests.tdml

```diff
@@ -304,7 +296,6 @@
 
     <!-- Same, but has minOccurs=maxOccurs for the arrays.
      That makes all array elements "required"
-
      treatAsMissing causes this to fail. Required empty non-defaultable is an error in that case.
```

Comment:
change comment to treatAsAbsent, and remove parenthetic note about fixing 2496.

Reply:
This is either updated or got revised out